src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1787,32 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitHandleWaitStart">
+                        <data name="WaitSource" inType="win:UInt32" map="WaitHandleWaitSourceMap" />
```

Comment:
Do we need 4 bytes for the wait source? Shouldn't a UInt8 be enough?

Reply:
I used that first but I saw other enum with greater sizes so I preferred to be safe here.
GCReasonMap -> 32 bits
GCSuspendEEReasonMap -> 16 bits
ThreadAdjustmentReasonMap -> 32 bits
TailCallTypeMap -> 32 bits
GCRootKindMap -> 32 bits

Reply:
Probably a byte would be enough, a byte would have been enough for `ThreadAdjustmentReasonMap` too, but it started off being 4 bytes so that can't be changed now.